src/Microsoft.DotNet.Build.Tasks/Targets/sign.targets

```diff
@@ -26,20 +26,14 @@
         * AfterTargets=PrepareForRun -> hit after @(IntermediateAssembly) has already been copied to output.
```

Comment:
Have a look at this comment when trying to decide how to hook in the post-processing targets.